nym-wallet/src-tauri/src/wallet_storage/mod.rs

```diff
@@ -152,6 +153,48 @@ pub(crate) fn store_login_with_multiple_accounts(
     store_login_with_multiple_accounts_at_file(&filepath, mnemonic, hd_path, id, password)
 }
 
+/// Update all logins with multiple accounts support
```

Comment:
You update all logins, whether each of them contain multiple accounts or not. The current wording makes it sounds like you're updating only the ones with multiple accounts support